src/Sulu/Bundle/AdminBundle/Resources/js/components/Backdrop/Backdrop.js

```diff
@@ -26,6 +33,14 @@ export default class Backdrop extends React.PureComponent {
             [backdropStyles.backdrop]: true,
             [backdropStyles.isVisible]: isVisible,
         });
+        const body = document.body;
```

Comment:
What exactly would we gain from moving this logic into `componentWillReceiveProps`? In this component every render is preceded by a call to `componentWillReceiveProps`, but not every `componentWillReceiveProps` is followed by a call to `render`. So there will be more unnecessary executions of this code when moving this to  `componentWillReceiveProps`. ()